CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to {{ project }}
+
+_NOTE: This is a template document that requires editing before it is ready to use!_
+
+We welcome contributions from the community and first want to thank you for taking the time to contribute!
+
+Please familiarize yourself with the [Code of Conduct](https://github.com/vmware/.github/blob/main/CODE_OF_CONDUCT.md) before contributing.
+
+_TO BE EDITED: Depending on the open source license that governs the project, leave only one of the options below:_
+
+* _DCO: Before you start working with {{ project }}, please read our [Developer Certificate of Origin](https://cla.vmware.com/dco). All contributions to this repository must be signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on as an open-source patch._
+* _CLA: Before you start working with {{ project }}, please read and sign our Contributor License Agreement [CLA](https://cla.vmware.com/cla/1/preview). If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ]([https://cla.vmware.com/faq](https://cla.vmware.com/faq))._
+
+## Ways to contribute
+
+We welcome many different types of contributions and not all of them need a Pull request. Contributions may include:
+
+* New features and proposals
+* Documentation
+* Bug fixes
+* Issue Triage
+* Answering questions and giving feedback
+* Helping to onboard new contributors
+* Other related activities
+
+## Getting started
+
+_TO BE EDITED: This section explains how to build the project from source, including Development Environment Setup, Build, Run and Test._
+
+_Provide information about how someone can find your project, get set up, build the code, test it, and submit a pull request successfully without having to ask any questions. Also include common errors people run into, or useful scripts they should run._
+
+_List any tests that the contributor should run / or testing processes to follow before submitting. Describe any automated and manual checks performed by reviewers._
+
+
+## Contribution Flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+* Make a fork of the repository within your GitHub account
+* Create a topic branch in your fork from where you want to base your work
+* Make commits of logical units
+* Make sure your commit messages are with the proper format, quality and descriptiveness (see below)
+* Push your changes to the topic branch in your fork
+* Create a pull request containing that commit
+
+We follow the GitHub workflow and you can find more details on the [GitHub flow documentation](https://docs.github.com/en/get-started/quickstart/github-flow).
+
+Before submitting your pull request, we advise you to use the following:

```

Comment:
Maybe this sentence fits better under ### Pull Request Checklist and can be truncated to "Before submitting pull request:"?  Overall it looks good to me!